src/components/table/table.tsx

```diff
@@ -205,6 +213,37 @@ export class Table {
         this.changePage.emit(page);
     }
 
+    private onClickRow(event: MouseEvent, row: Tabulator.RowComponent): void {
+        // deselect all activated rows
+        this.tabulator.deselectRow();
+        row.select();
```

Comment:
We can not actually select the row in the table like this, since that will conflict when we actually want to select rows later. This is not a "select" functionality, just an "activate" functionality

Reply:
* a "selected" row (using checkboxes) can be "activated" a well, and activating shouldn't deselect it.
* multiple rows can be "selected"
* only 1 row at a time can be "activated". Activating a row must deactivate any other row that is currently active